httpie/plugins/__init__.py

```
@@ -0,0 +1,24 @@
+"""
+WARNING: The plugin API is still work in progress and will
+         probably be completely reworked by v1.0.0.
+
+"""
+from httpie.plugins.base import (
+    AuthPlugin, FormatterPlugin,
```

Ruff (0.8.2)

`httpie.plugins.base.AuthPlugin` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)